README.md

```diff
@@ -11,6 +11,7 @@
 [5]: https://ci.appveyor.com/api/projects/status/0bf9glha2yg9x8ef/branch/master?svg=true
 [6]: https://ci.appveyor.com/project/spdx/tools-python/branch/master
 
+
```

Comment:
Remove this file diff if no changes here.

Reply:
thanks. removed it..